makefile

```diff
@@ -1,26 +1,27 @@
 # Helper function to perform recursive search in a directory for file types.
```

Comment:
I don't know enough about makefiles to review this lol. We can maybe go over it in person when we next sync up.